### PR TITLE
Adapt automation to consider the results from all previous iterations…

### PR DESCRIPTION
**Pull request by parnumeric**
… instead of only the last run

### CODE REVIEW

**Comment by parnumeric** [comment]
The $MERGE_FILE to store all previous concatenated merged_mses.csv need must not be touched in advance, because the program will fail reading the empty file. So, the $MERGE_FILE should not exist before the simulation.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/psf/black
-    rev: 20.8b1 # Replace by any tag/version: https://github.com/psf/black/tags
+    rev: 22.3.0 #20.8b1 # Replace by any tag/version: https://github.com/psf/black/tags
```
Comment (alisonrclarke) [inline .pre-commit-config.yaml:3]: Can you update the other references to black to match this build number, in the conda files and in the GitHub workflow? (See fba532e for last time I changed the versions.)

**Comment by alisonrclarke** [comment]
> The `$MERGE_FILE` to store all previous concatenated `merged_mses.csv` must not be touched in advance, because the program will fail reading the empty file. In fact, it should be removed if exists. So, the `$MERGE_FILE` must not exist before the simulation.

Can you add this detail to the Hamilton README file please? Or could the script delete it automatically, or fail if it exists?

**Comment by alisonrclarke** [comment]
Build failure was because secrets aren't available in @parnumeric's copy of the repo. #150 is a replacement branched from the DurhamARC org, so closing this PR.

**Pull request closed without merging**